bundle/curl.sh

```diff
@@ -2,7 +2,7 @@
 
 set -ex
 
-curl https://curl.haxx.se/download/curl-7.64.0.tar.gz | tar zx
-cd curl-7.64.0
-./configure  --enable-optimize --disable-dict --disable-gopher --disable-ftp --disable-imap --disable-ldap --disable-ldaps --disable-pop3 --disable-proxy --disable-rtsp --disable-smtp --disable-telnet --disable-tftp --disable-zlib --without-ca-bundle --without-gnutls --without-libidn --without-librtmp --without-libssh2 --without-nss --without-ssl --without-zlib --disable-shared  --enable-threaded-resolver
+curl https://curl.haxx.se/download/curl-7.64.1.tar.gz | tar zx
```

Comment:
Why not bump to the latest, `7.65.0`?

Reply:
I got an error compiling that version and I didn't want to track it down:

```url.c: In function ‘Curl_init_userdefined’:
url.c:443:6: error: ‘struct UserDefined’ has no member named ‘ftp_use_epsv’
   set->ftp_use_epsv = TRUE;   /* FTP defaults to EPSV operations */
      ^~
url.c:444:8: error: ‘struct UserDefined’ has no member named ‘ftp_use_eprt’; did you mean ‘use_port’?
   set->ftp_use_eprt = TRUE;   /* FTP defaults to EPRT operations */
        ^~~~~~~~~~~~
        use_port
url.c:445:8: error: ‘struct UserDefined’ has no member named ‘ftp_use_pret’; did you mean ‘use_port’?
   set->ftp_use_pret = FALSE;  /* mainly useful for drftpd servers */
        ^~~~~~~~~~~~
        use_port
url.c:446:6: error: ‘struct UserDefined’ has no member named ‘ftp_filemethod’
   set->ftp_filemethod = FTPFILE_MULTICWD;
      ^~
```

Reply:
My guess is that since we explicitly disable ftp support (which is enabled by default) they are not dealing with that configuration properly.